testing/examples/extended/BUILD

```diff
@@ -64,3 +64,87 @@ container_test(
     driver = "tar",
     image = "@extended_alpine_dockerfile//image:dockerfile_image",
 )
+
+# Test the OCI image option of new_container_pull
+container_image(
```

Comment:
Not need to add examples in testing/examples. Just the basic cases in tests/container/BUILD.